docs/_jul_milestone_.md

```diff
@@ -0,0 +1,60 @@
+# Monthly Milestone - July 2020
+
+## Overview
+
+Focus on:
+
+- documentation improvements
+- reduce backlog of PRs waiting to be reviewed
+
+## Goals
+
+- [ ] Docs improvements #5113
+
+  - [ ] Hide @loopback/openapi-v3 from docs, examples and CLI templates #5692
+  - [ ] Docs: how to configure a custom table/collection name different from
+        model class name #2197
+  - [ ] Improve and refactor the authentication documentations #5294
+
```

Comment:
Can we add the following task?

- [Docs] Add "How the documentation is organized" section #5752